gallery/workbook/businesscontinuity-insights.json

```diff
@@ -0,0 +1,47 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/gallery.json",
+	"version": "TemplateGallery/1.0",
+	"categories": [
+		{
+			"id": "Azure Backup",
+			"name": "Azure Backup",
+			"templates": [
+				{
+					"id": "Workbooks/Azure Backup/Backup Report",
+					"author": "Microsoft",
+					"description": "Gain visibility into backup job statistics, consumption trends and optimization opportunities across your Recovery Services vaults and Backup vaults",
+					"name": "Backup Reports (consolidated)",
+          "isPreview": true
```

Comment:
Can you format all these "isPreview": true fields? They are all strangely indented